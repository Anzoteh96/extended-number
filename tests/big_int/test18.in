-8744543968707768065159351113537811908710764180549237187
382174505971130707745350478307796098604130186981052390
