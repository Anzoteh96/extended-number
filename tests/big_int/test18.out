Enter two numbers n and m:
n is -8744543968707768065159351113537811908710764180549237187
m is 382174505971130707745350478307796098604130186981052390
n + m is -8362369462736637357414000635230015810106633993568184797
n - m is -9126718474678898772904701591845608007314894367530289577
n * m is -3341941771183721923198593465107623266866039633282195486237347699149921441464928802043601587994647436983226930
n / m is -23
n % m is 45469668628238212983709887541498359184230120014967783
